common/api/core-backend.api.md

```diff
@@ -2680,18 +2680,30 @@ export interface ExportGraphicsMesh {
     points: Float64Array;
 }
 
+// @public
+export class ExportGraphicsMeshSubsetVisitor extends ExportGraphicsMeshVisitor {
+    static createSubsetVisitor(polyface: ExportGraphicsMesh, facetIndices: number[], numWrap?: number): ExportGraphicsMeshSubsetVisitor;
+    getVisitableFacetCount(): number;
+    moveToNextFacet(): boolean;
+    moveToReadIndex(subsetIndex: number): boolean;
+    parentFacetIndex(subsetIndex?: number): number | undefined;
+    reset(): void;
+}
+
 // @public
 export class ExportGraphicsMeshVisitor extends PolyfaceData implements PolyfaceVisitor {
+    protected constructor(facets: ExportGraphicsMesh, numWrap: number);
```

Comment:
Adding a protected constructor replaces the former default public no-arg constructor. That's a breaking change to a `@public` API.

Reply:
Currently, this constructor is private; is there a public no-arg constructor? I elevated it to protected so I could subclass.

Reply:
Never mind, extract-api doesn't include private constructor in its output so it looked like it was newly added, not changed.